rs/bitcoin/ckbtc/minter/ckbtc_minter.did

```diff
@@ -186,10 +186,16 @@ type InitArgs = record {
     /// The minter's operation mode.
     mode : Mode;
 
-    /// The fee paid per check by the KYT canister.
+    /// The fee paid per bitcoin check.
+    check_fee : opt nat64;
+
+    /// The fee paid per check by the KYT canister (depreicated, use check_fee instead).
     kyt_fee : opt nat64;
```

Comment:
AFAIK we do not need to be backwards compatible for init/upgrade args (it's not checked by the compatibility test) so I would suggest just removing the deprecated fields. WDYT?

Reply:
The problem is that events shares the same init and upgrade args. I actually tried to make separate types for  events, but find that the duplication is not ideal.

Reply:
> The problem is that events share the same init and upgrade args.

Good point, I had forgotten about this. It's unfortunate that the same type is used for the event and the Candid interface but I also don't have the impression that changing this is worth it at this point.

nit: there is a typo L.192 (`depreicated` -> `deprecated`)

Reply:
Thanks! Typo fixed